toolz/functoolz.py

```diff
@@ -541,3 +541,32 @@ def do(func, x):
     """
     func(x)
     return x
+
+
+@curry
+def flip(func, a, b):
+    """Call the function call with the arguments flipped.
+
+    Parameters
+    ----------
+    f : callable
+        A binary function.
+    a : any
+        The second argument to `f`.
+    b : any
+        The first argument to `f`.
+
+    Returns
```

Comment:
There is no need to use numpy-style documentation.  `toolz` documentation should be more concise, because the goal is that functions are simple enough that they can be described with kinder prose.